about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./app/assets/head.png" type="image/x-icon" />
+    <script
+      src="https://kit.fontawesome.com/bc94a42a6b.js"
+      crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="dist/style.css" />
+    <title>Smart | About Us</title>
+  </head>
+  <body>
+    <div class="container-about">
+      <!-- ================== Header ====================== -->
+
+      <header class="header-top flex flex-jc-b flex-center limit">
+        <p class="logo flex flex-center">
+          <img src="./app/assets/head.svg" alt="" />
+          <a href="index.html">Sm<span class="span">art</span></a>
+        </p>
+        <nav class="menu flex flex-al-c flex-jc-c">
+          <ul class="nav-links flex flex-jc-b">
+            <li><a class="nav-link" href="index.html">Home</a></li>
+            <li><a class="nav-link" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#program">Program</a></li>
+            <li><a class="nav-link" href="#join">Join</a></li>
+            <li><a class="nav-link" href="#sponsor">Sponsor</a></li>
+            <li><a class="nav-link contact" href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+
+        <div class="btn-container flex flex-center">
+          <span class="menu-btn"></span>
+        </div>
```

Comment:
- [ ] Please note that you are to stick to design as much as possible while personalizing it. Kindly move your hamburger button to the left of the mobile page.